docs/conf.py

```diff
@@ -47,7 +47,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "OpenTelemetry Python"
-copyright = "OpenTelemetry Authors"  # pylint: disable=redefined-builtin
+docs_copyright = "OpenTelemetry Authors"  # pylint: disable=redefined-builtin
```

Comment:
I am pretty sure we cannot change sphinx configuration for pleasing some lint rules :sweat_smile:  see
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright

Reply:
Can ignore this too